pkg/venafi/cloud/connector.go

```diff
@@ -25,6 +25,12 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
+	"github.com/Khan/genqlient/graphql"
```

Comment:
Wrong spot for these? Probably won't pass the linter...

Reply:
We don't have lint rules to check the goimports in here. But yeah, it's good practice to follow them now. Good catch

Reply:
Updated by latest commit